### PR TITLE
test user SSH keys

### PR DESCRIPTION
**Pull request by zreigz**
**What this PR does / why we need it**: test user SSH keys

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

```release-note
NONE
```

### CODE REVIEW

File: api/pkg/test/e2e/api/admin_test.go [hunk 1]
```diff
@@ -398,3 +399,65 @@ func TestManageClusterByAdmin(t *testing.T) {
...
+			name:      "admin can manage SSH keys for any project",
+			keyName:   "test",
+			publicKey: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC8LlXSRW4HUYAjzx1+r5JzpjXIDDyFkWZzBQ8aU14J8LdMyQsU6/ZKuO5IKoWWVoPi0e63qSjkXPTjnUAwpE62hDm6uLaPgIlc3ND+8d9xbItS+gyXk9TSkC3emrsCWpS76W3KjLwyz5euIfnMCQZSASM7F5CrNg6XSppOgRWlyY09VEKi9PmvEDKCy5JNt6afcUzB3rAOK3SYZ0BYDyrVjuqTcMZwRodryxKb/jxDS+qQNplBNuUBqUzqjuKyI5oAk+aVTYIfTwgBTQyZT7So/u70gSDbRp9uHI05PkH60IftAHdYu4TJTmCwJxLW/suOEx3PPvIsUP14XQUZgmDJEuIuWDlsvfOo9DXZNnl832SGvTyhclBpsauWJ1OwOllT+hlM7u8dwcb70GD/OzCG7RSEatVoiNtg4XdeUf4kiqqzKZEqpopHQqwVKMhlhPKKulY0vrtetJxaLokEwPOYyycxlXsNBK2ei/IbGan+uI39v0s30ySWKzr+M9z0QlLAG7rjgCSWFSmy+Ez2fxU5HQQTNCep8+VjNeI79uO9VDJ8qvV/y6fDtrwgl67hUgDcHyv80TzVROTGFBMCP7hyswArT0GxpL9q7PjPU92D43UEDY5YNOZN2A976O5jd4bPrWp0mKsye1BhLrct16Xdn9x68D8nS2T1uSSWovFhkQ== lukasz@loodse.com ",
+		},
+	}
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			masterToken, err := retrieveMasterToken()
+			if err != nil {
+				t.Fatalf("can not get master token due error: %v", err)
```
Comment (lsviben) [inline api/pkg/test/e2e/api/admin_test.go]: NIT: `due error` -> `due to error` here and below

File: api/pkg/test/e2e/api/ssh_test.go [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+			}
+			teardown := cleanUpProject(project.ID, 1)
+			defer teardown(t)
+
+			sshKey, err := apiRunner.CreateUserSSHKey(project.ID, tc.keyName, tc.publicKey)
+			if err != nil {
+				t.Fatalf("can not get create SSH key due error: %v", err)
+			}
+			// wait for the controller to setup RBACs
+			time.Sleep(15 * time.Second)
```
Comment (lsviben) [inline api/pkg/test/e2e/api/ssh_test.go]: instead of sleep, could we actively poll the api for the wanted RBACs until they are ready, with some timeout?

**Review by lsviben — CHANGES_REQUESTED** [review]

File: api/pkg/test/e2e/api/ssh_test.go (hunk 2 above)
Comment (zreigz) [reply api/pkg/test/e2e/api/ssh_test.go]: yes, I can do this. We have a similar approach in other tests

**Pull request merged**